pkg/conference/matrix_message_processor.go

```diff
@@ -68,6 +68,7 @@ func (c *Conference) onNewParticipant(participantID ParticipantID, inviteEvent *
 				}) == nil
 			},
 			OnTimeout: func() {
+				c.logger.Debug("We haven't received an m.call.ping from the client for more than the timeout")
```

Comment:
Note, we probably also don't necessarily need to log it here, because when the `LeftTheCall` message with the keep alive timeout reason is sent in a line below, it gets logged already. Currently, the reason is `Hangup` though, so it's the `LeftTheCall` that we get from the connection state change below, not from the timeout.